pkgs/servers/sql/postgresql/buildPostgresExtension.nix

```diff
@@ -0,0 +1,66 @@
+{
+  stdenv,
+  postgresql,
+}:
+
+args:
+(stdenv.mkDerivation args).overrideAttrs (prevAttrs: {
+  buildInputs = [ postgresql ] ++ prevAttrs.buildInputs or [ ];
+
+  installFlags = [
+    "DESTDIR=${placeholder "out"}"
+  ] ++ prevAttrs.installFlags or [ ];
+
+  postInstall =
+    ''
+      # DESTDIR + pg_config install the files into
+      # /nix/store/<extension>/nix/store/<postgresql>/...
+      # We'll now remove the /nix/store/<postgresql> part:
+      if [[ -d "$out${postgresql}" ]]; then
+          cp -arlt "$out" "$out${postgresql}"/*
+          rm -r "$out${postgresql}"
+      fi
+
+      if [[ -d "$out${postgresql.dev}" ]]; then
+          mkdir -p "''${dev:-$out}"
+          cp -arlt "''${dev:-$out}" "$out${postgresql.dev}"/*
```

Comment:
What's the reason for using `-t` rather than putting the target dir last (after the glob for the sources)? Is `-t` common/known enough that readers will recognize it (I had to look it up), or would it be better to use the "long" option `--target-directory`?